frontend/src/entry/model/releases/FileDownload.tsx

```diff
@@ -22,8 +26,9 @@ interface ChipDetails {
   icon: ReactElement
 }
 
-export default function FileDownload({ modelId, file }: FileDownloadProps) {
+export default function FileDownload({ modelId, file, hideAssociatedReleases = true }: FileDownloadProps) {
```

Comment:
It's standard practice to default boolean props to `false` if they're optional. I'd suggest following convention and adjusting the props where this component is called.